Tests/CreateAPITests/Specs/edgecases.yaml

```
@@ -1001,6 +1003,15 @@ components:
         color:
           type: string
           default: red
+    Image:
+      type: object
+      required:
+        - id
+        - url
+      id:
+        - type: string
+      url:
+        - type: string
```

I think the yaml formatting is a little off here which is why the `type` is being ignored for the generated `Image` (it's `AnyJSON` when it should be `String`)

iirc, the `-` denotes an array item, so instead we want this:

```suggestion
    Image:
      type: object
      required:
        - id
        - url
      id:
        type: string
      url:
        type: string
```